drivers/motor/foc/Make.defs

```diff
@@ -30,4 +30,4 @@ endif
 
 DEPPATH += --dep-path motor$(DELIM)foc
 VPATH += :motor$(DELIM)foc
-CFLAGS += ${shell $(INCDIR) $(INCDIROPT) "$(CC)" $(TOPDIR)$(DELIM)drivers$(DELIM)motor$(DELIM)foc}
\ No newline at end of file
+CFLAGS += ${shell $(INCDIR) "$(CC)" $(TOPDIR)$(DELIM)drivers$(DELIM)motor$(DELIM)foc}
```

Comment:
keep the last \n

Reply:
The last \n is there, just not displayed